Source/IGListStackedSectionController.h

```diff
@@ -25,6 +26,9 @@ IGLK_SUBCLASSING_RESTRICTED
 NS_SWIFT_NAME(ListStackedSectionController)
 @interface IGListStackedSectionController : IGListSectionController
 
+/// A bitmask of experiments to conduct on the section controller.
```

Comment:
For one liner comment, could you also use the following style to keep the doc consistent? thanks!

/**
 <your_comment_goes_here>
 */